manipulation/src/functTest/java/org/jboss/gm/manipulation/SimpleProjectFunctionalTest.java

```diff
@@ -54,10 +58,10 @@ public void ensureProperPomGenerated() throws IOException, URISyntaxException, X
 
         final ManipulationModel alignment = ManipulationIO.readManipulationModel(simpleProjectRoot);
 
-        final BuildResult buildResult = GradleRunner.create()
+        final BuildResult buildResult = TestUtils.createGradleRunner()
                 .withProjectDir(simpleProjectRoot)
-                .withArguments("generatePomFileForMainPublication")
-                .withDebug(true)
+                .withArguments("--info", "generatePomFileForMainPublication")
```

Comment:
Do we want `--info`?

Reply:
*shrug* Fair point, I could remove it. Makes very little difference and was left in by mistake when debugging.

Reply:
No problem 😊